homeassistant/components/dsmr/config_flow.py

```diff
@@ -346,6 +348,12 @@ async def async_step_init(
                             CONF_TIME_BETWEEN_UPDATE, DEFAULT_TIME_BETWEEN_UPDATE
                         ),
                     ): vol.All(vol.Coerce(int), vol.Range(min=0)),
+                    vol.Optional(
+                        CONF_RECONNECT_INTERVAL,
+                        default=self.entry.options.get(
```

Comment:
`default=self.options.get(CONF_RECONNECT_INTERVAL, self.data.get(CONF_RECONNECT_INTERVAL, DEFAULT_RECONNECT_INTERVAL))`

Reply:
A bit ugly, but for imported configs, the value is in the data, not in the options.